docs/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM nikolaik/python-nodejs:python3.8-nodejs18
```

Comment:
We need `node` to build the HTML files of the docs. Not sure if this is the best way to have both Python and Node in the same build